File: week07/count_word.pl
#!/usr/bin/perl -w

$word = lc $ARGV[0];
$count = 0;

while($line = <STDIN>) {
   undef @words;
   $line = lc $line;

   @words = ($line =~ /\b$word\b/g);
   $count += @words;
}

print "$word occurred $count times\n";

File: week07/count_word.py
#!/usr/bin/python3

import sys
import re

count = 0
word = sys.argv[1].lower()
for line in sys.stdin:
   words = []
   line = line.lower()
   words = re.findall(r'\b'+word+r'\b', line)
   count += len(words)

print("%s occurred %d times" %(sys.argv[1], count))
